03-attribute-operations.Rmd

```diff
@@ -584,7 +531,7 @@ Hint: try to use helper functions, such as `contains` or `starts_with` from **dp
 6. What was the minimum and maximum total population in 2015 in each region? What was the total population in 2015 in each region?
 <!-- Attribute joining -->
 7. Add variables from `us_states_df` to `us_states` and create a new object called `us_states_stats`. What function did you use and why? Which variable is the key in the both datasets? What is the class of a new object?
-8. `us_states_df` has two more variables than `us_states`. How you can find them?
+8. `us_states_df` has two more variables than `us_states`. How you can find them? (hint: try to use the `dplyr::anti_join` function)
```

Comment:
That's a good challenge. Just wondering if there's another question we could ask about right joins but nothing comes to mind.